domain/apiresponses/failure_responses.go

```diff
@@ -16,6 +16,7 @@ type FailureResponse struct {
 	loggerAction  string
 	emptyResponse bool
 	errorKey      string
+	errorKeyBool  bool // needed for instance_usable and update_repeatable
```

Comment:
- We think having a single fields is to reduce complexity for users. We were concerned that it reduces the flexibility in certain situations. For example, what if an update fails and `update_repeatable` should be false, but `instance_usable` should be true? We think it would actually be better to split out these fields so that users have control over which are set.
- This field is currently private which means that it can't be set by code outside of this package. We think that `NewFailureResponse()` may need to be extended. One way of doing this would be the functional option pattern like this: https://github.com/pivotal-cf/brokerapi/blob/697e71d190b9f83735ac2bbaa4dacb430b4952c1/api_options.go#L28

Reply:
thank you, I have another look